samples/modules/mctp/mctp_endpoint/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+
+project(i2c_peripheral)
```

Comment:
Wrong name?

Reply:
Fixed this and the mctp host project name as well (was also wrong)